Source/dead.cpp

```diff
@@ -16,13 +16,14 @@ DeadStruct Dead[MaxDead];
 int8_t stonendx;
 
 namespace {
-void InitDeadAnimationFromMonster(DeadStruct &dead, const CMonster &mon)
+void InitDeadAnimationFromMonster(DeadStruct &dead, CMonster &mon)

```

Comment:
Providing a const version of CMonster::GetAnimData should allow keeping mon as a const ref

Reply:
I got a error in `AnimationInfo` and thought I would need to update many places.
But after trying it, it wasn't many places that hand to be adjusted.
So I added it (last commit) 👍🙂